README.md

```diff
@@ -168,7 +168,7 @@ shoryuken [options]
     -c, --concurrency INT            Processor threads to use
     -d, --daemon                     Daemonize process
     -q, --queue QUEUE[,WEIGHT]...    Queues to process with optional weights
-    -r, --require [PATH|DIR]         Location of the worker
+    -r, --require [PATH]             Location of the worker
```

Comment:
@rubemz seems right! I was wondering if it would be better to change [this](https://github.com/phstc/shoryuken/blob/555953fb5e4c37933335109adc1e23d5197c382b/lib/shoryuken/environment_loader.rb#L156) instead to check `File.directory?` and load all files inside a directory in case of a directory is supplied? WDYT?